Foundation/FileHandle.swift

```diff
@@ -888,6 +893,10 @@ extension FileHandle {
     }
 
     open func acceptConnectionInBackgroundAndNotify(forModes modes: [RunLoop.Mode]?) {
+#if os(Windows)
```

Comment:
Since this is for treating fds like sockets, it's fine if this is unavailable on Windows only.